builds.sh

```diff
@@ -19,15 +19,27 @@ shell="$(jq -r '
 ' "$json")"
 eval "$shell"
 
+_resolveRemoteArch() {
+	local img="$1"; shift
+
+	local arches
+	if ! arches="$(bashbrew remote arches --json "$img" 2>/dev/null)"; then # TODO somehow differentiate errors like 404 / 403, "insufficient_scope" from other bashbrew errors here so we can stop eating stderr
+		printf 'null'
+		return
+	fi
+
+	echo "$arches"
+}
+
 # resolve an image reference to an architecture-specific imageId (digest of the image manifest)
 _resolve() {
 	local img="$1"; shift
 	local arch="$1"; shift
+	local bashbrewRemoteArches="$1"; shift
 
-	local bashbrewRemoteArches
-	if ! bashbrewRemoteArches="$(bashbrew remote arches --json "$img" 2>/dev/null)"; then # TODO somehow differentiate errors like 404 / 403, "insufficient_scope" from other bashbrew errors here so we can stop eating stderr
-		printf 'null'
-		return
+	if [ "$bashbrewRemoteArches" = 'null' ]; then
+		printf 'null';
+		return;
```

Comment:
This must be my bad for giving you a one-liner example: :sweat_smile:

```suggestion
		printf 'null'
		return
```

Reply:
It's also a habit for me doing image commands one-liners

Reply:
If we're removing unnecessary characters, there is no need to quote a one-word literal.

Reply:
I'm just going on consistency with the other similar function that does this